frontend/src/component/project/Project/ProjectInfo/ChangeRequestsWidget.tsx

```diff
@@ -11,6 +11,8 @@ import {
 import { useProjectChangeRequests } from 'hooks/api/getters/useProjectChangeRequests/useProjectChangeRequests';
 import { WidgetFooterLink } from './WidgetFooterLink';
 
+const LOADING_LABEL = 'change-requests-widget';
```

Comment:
why do we need custom selector here?

Reply:
Data for change requests is being loaded from another endpoint (that isn't available in unleash-core)